Test.cmd

```diff
@@ -15,12 +19,6 @@ call Ensure-DotNetSdk.cmd
 pushd "%CMDHOME%"
 @cd
 
-SET TestResultDir=%CMDHOME%\Binaries\%BuildConfiguration%\TestResults
```

Comment:
Why delete this? I believe it is used by the ADO build.

Reply:
This was the value passed as the $outDir param in `Parallel-Tests.ps1`, which isn't used in that script. 
Sounds like you are saying this is read as an environment variable by something in one of the builds?
Would you prefer I can leave the EV even though the param is not used?

Reply:
I un-removed these EV's, although I can't find where they might be used.